Add local EBT shipper (#189671)
elastic/kibana: examples/ebt_local_shipper/README.md

## EBT local shipper

This example plugin registers a local shipper that indexes all the EBT events in the local Elasticsearch.

> The intention is to make it easier for developers to explore their events without having to wait for our _long_ ingestion pipelines. 

Reviewer: ```suggestion
The intention is to allow developers to bypass ingestion pipelines and explore their events locally. 
```

Reviewer: Hopefully, ingestion pipelines will speed up.
